generators/client/templates/angular/src/main/webapp/app/layouts/error/error.route.ts.ejs

```diff
@@ -37,5 +37,18 @@ export const errorRoute: Routes = [
             pageTitle: 'error.title',
             error403: true
         },
+    },
+    {
+        path: '404',
+        component: ErrorComponent,
+        data: {
+            authorities: [],
+            pageTitle: 'error.title',
+            error404: true
+        },
+    },
+    {
+        path: '**',
+        redirectTo: '/404',
```

Comment:
Thanks a lot, I forgot about this. 
Do you think it's a better thing to move the import of AppRoutingModule at the bottom of the module imports of app.module.ts, or do you think it's more explicit to directly create a new module just for the 404 handling at the bottom of the imports?

Reply:
The Angular guide puts the wildcard route in `AppRoutingModule` and imports `AppRoutingModule` after other modules, so that's probably the way to do it https://angular.io/guide/router#module-import-order-matters (search for the heading `Module import order matters`)